Mathlib/Analysis/SpecialFunctions/Trigonometric/Basic.lean

```diff
@@ -759,6 +759,21 @@ theorem cos_pi_div_three : cos (π / 3) = 1 / 2 := by
       refine cos_lt_cos_of_nonneg_of_le_pi ?_ le_rfl ?_ <;> linarith [pi_pos]
     linarith [cos_pi]
 
+/-- The cosine of `2 * π / 3` is `- 1 / 2`. -/
+@[simp]
+theorem cos_two_mul_pi_div_three : cos (2 * π / 3) = - 1 / 2 := by
+  rw [mul_div_assoc, cos_two_mul, cos_pi_div_three]
+  norm_num
+
+/-- The cosine of `4 * π / 3` is `- 1 / 2`. -/
+@[simp]
+lemma cos_four_mul_pi_div_three : cos (4 * π / 3) = - 1 / 2 := calc
```

Comment:
nit: I'm not sure this is an established style, and would recommend putting the `calc` on the next line and indenting everything below it one more level.